cpp/ps0002/ps0002_Solution.cpp

```diff
@@ -1,50 +1,27 @@
-
 class ps0002_Solution
 {
-
-    /* Give examples for variables from real life and consider that:
-    - variables should be named in a way that makes sense
-    - variables should has a suitable data type
-    - variables should be initialized with a suitable value
-    - example:
-        int family_members = 8{};
-*/
-
-
-
-
     /*        
     أي احسن نوع بيانات تختاره لمتغير هيشيل قيمة:
-- عدد سكان عمارة
-- عدد البدايل لدوا في الصيدلية
-- أفراد تيم في شركة برمجة
-- سعر واحدة من مخبوزات الفرن
-- الحرف الأول من اسم شخص
-- عدد المواليد في دولة تعدادها كبير زي الصين
-- ناتج اكثر دقة لقسمة رقم عىل 7
-- فاتورة الكهربا لخط انتاج في مصر
-- عمر أشخاص كبار و صغار ف السن بالثانية    
-    
+    - عدد سكان عمارة
+    - عدد البدايل لدوا في الصيدلية
+    - أفراد تيم في شركة برمجة
+    - سعر واحدة من مخبوزات الفرن
+    - الحرف الأول من اسم شخص
+    - عدد المواليد في دولة تعدادها كبير زي الصين
+    - ناتج اكثر دقة لقسمة رقم عىل 7
+    - فاتورة الكهربا لخط انتاج في مصر
+    - عمر أشخاص كبار و صغار ف السن بالثانية    
     */
 
-
-
-
-    int building_residents_count{40};
-    int medicine_alternatives{7};
-    int development_team_count{15};
-
-    float unti_price{15.50f};
-    char first_letter{'a'};
-    long int population_count{ 1344130000 };
-    float divide_result{ 3.5714285714285714285714285714286f };
-    float electricity_bill{150.45f};
-    long int age_in_seconds{ 1281657600 };
-
-
-
-
-
-
+    int building_residents_count{40}; // 0.5/0.75
+    int medicine_alternatives{7}; // 0.75/0.75
+    int development_team_count{15}; // 0.75/0.75
+    float unti_price{15.50f}; // 0.75/0.75
+    char first_letter{'a'}; // 0.75/0.75
+    int population_count{ 1344130 }; // 0.25/0.75
+    double divide_result{ 3.5714285714285714285714285714286 }; // 0.25/0.75
+    float electricity_bill{150.45f}; // 0.75/0.75
+    unsigned int age_in_seconds{ 1281657600 }; // 0.5/0.75
+    // total 5.25/6.75
 };
-
+// great work 😎 9/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code
+0.5 bonus mark for using `class` and correct literals `f`